src/settings.py

```diff
@@ -0,0 +1,208 @@
+"""
+Django settings for src project.
+
+Generated by 'django-admin startproject' using Django 2.2.16.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.2/ref/settings/
+"""
+
+import os
+import dj_database_url
+import dotenv
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+dotenv_file = os.path.join(BASE_DIR, ".env")
+if os.path.isfile(dotenv_file):
+    dotenv.load_dotenv(dotenv_file)
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = os.environ.get('SECRET_KEY')
```

Comment:
Do not use `os.environ.get(foo)` as this will return `None` if no variable is set. Instead use `os.environ[foo]` otherwise, TypeError's can occur such as on [src/urls.py:20](https://github.com/F4MP/CloudService/pull/1/files/2f2cb02e3e2823ea58d37b6217de906e4f880aa6#diff-5856bc1079f62af1013cb03a644f1e54R20) where it will raise for concatenating None with a str